committee-steering/governance/annual-reports.md

```diff
@@ -0,0 +1,125 @@
+modified timeline bootstrap process this year and then will follow the process
+outlined in 2021. This means that Chairs will be reporting twice in n months and
+not a year. A yearly cadence will be followed post 2021.
+
+TODO outline time line for 2020  
+
+### 2021 and on:
+1. Chairs create one report and send it to steering-private@kubernetes.io for
+review; reports are due at the end of February every year and can pick a
+collaboration doc tool of their choice (google docs, hackmd, etc)
+2. Steering Committee will assign one member to each community group before the
+reports are submitted so there is an established point of contact for any help or
+guidance needed
+3. The March SIG Chairs and Tech Leads meeting will be used as follow up and
+report retrospectives
+4. Steering will schedule private meetings to follow up on anything denoted as
+a private matter on the report; reports are not made public until all private
+comments have been addressed
+5. Once reviewed and follow ups are completed, Steering will ask the chair(s)
+of the group to submit a pull request to that groups documentation in the
+community repo.
+6. The Steering Committee liaison will work directly with groups that have follow
+up items and update Steering during regular monthly meetings
+
+### Tips for Chairs:  
+- SIG Chairs are responsible for collecting and reporting information for
+sponsored Working Groups.    
+- Respond with n/a to questions that are not applicable.  Most questions will be.
+- Please include any and all private matters between [private] tags [/private].
+The published report will not include any private matters. All reports are treated
+as private until Steering asks for a PR, which will not include private tags.
+- Wait until Steering gives you the go ahead before submitting a pull request.  
+[TODO what else?]
+
+## Questions for report:
+
+Copy these questions as a template with the applicable sections   
+
+#### You and your role:
+- Do you enjoy the role? What do you find challenging? How can we/is there
+something we can provide that would better support you? Do you want to continue
+or find a replacement?
+- Do you have goals for the group? If you feel that you aren’t ready to pass the
+baton, what would you like to accomplish before you do?
+- How can Steering help? Do you have feedback for us? Suggestions for what we
+should work on?
+
+#### Special Interest Groups:
+- How are you doing with operational tasks in [yourgroup]-governance.md?
+- Is your README accurate? have a CONTRIBUTING.md file?
+- All subprojects correctly mapped and listed in sigs.yaml?
+- Are all subproject maintained? Are there any that are being retired?
+- Tell us about your meetings: large/small, active/quiet? learnings? up to date
+on the Kubernetes community YouTube under a community group playlist? Meeting
+notes up to date?
+- When was your last public community update? (provide link)
+
+##### Membership
```

Comment:
Can we also ask if they have a company diversity in SIG members i.e. contributors are from multiple companies (we do that below for WGs, we could do that for SIGs as well)?